benchmarks/python/test_transformer.py

```diff
@@ -979,19 +1023,19 @@ def transformer_backward_fusion(fd: FusionDefinition) -> None:
     T454 = fd.ops.add(T384, T453)
     T455 = fd.ops.add(T304, T454)
     T456 = fd.ops.cast(T455, dtype=DataType.BFloat16)
-    fd.add_output(T184)
-    fd.add_output(T186)
-    fd.add_output(T223)
-    fd.add_output(T225)
-    fd.add_output(T228)
-    fd.add_output(T232)
-    fd.add_output(T324)
-    fd.add_output(T326)
-    fd.add_output(T373)
-    fd.add_output(T376)
-    fd.add_output(T379)
-    fd.add_output(T383)
-    fd.add_output(T456)
+    fd.add_output(T184) # MLP linear1.weight_grad
+    fd.add_output(T186) # MLP linear1.bias_grad
+    fd.add_output(T223) # MLP linear0.weight_grad
+    fd.add_output(T225) # MLP linear0.bias_grad
+    fd.add_output(T228) # layer_norm1.bias_grad
+    fd.add_output(T232) # layer_norm1.weight_grad
+    fd.add_output(T324) # MHA linear1.weight_grad
+    fd.add_output(T326) # MHA linear1.bias_grad
+    fd.add_output(T373) # MHA linear0.weight_grad
+    fd.add_output(T376) # MHA linear0.bias_grad
+    fd.add_output(T379) # layer_norm0.bias_grad
+    fd.add_output(T383) # layer_norm0.weight_grad
+    fd.add_output(T456) # dy output grad
```

Comment:
Is this "dx"? Otherwise, how is it different from "dy", T20?

Reply:
Good catch, it's dx!